app/destination/page.js

```diff
@@ -1,54 +1,74 @@
+    description:
+      "Europa, one of Jupiter’s moons, is an icy world with a hidden ocean beneath its surface. This mysterious moon is a prime candidate for the search for extraterrestrial life, making it a thrilling destination for space explorers.",
+    thumbnail: "/destination/image-europa.png",
+    isSelected: false,
+  },
+  {
+    name: "MARS",
+    description:
+      "Mars, the Red Planet, is a barren yet fascinating world with vast deserts, towering volcanoes, and the deepest canyon in the solar system. As humanity’s next frontier, Mars invites us to dream of colonization and the possibilities of life beyond Earth.",
+    thumbnail: "/destination/image-mars.png",
+    isSelected: false,
+  },
+  {
+    name: "MOON",
+    description:
+      "Our closest celestial neighbor, the Moon, is a silent witness to Earth's history. With its stunning craters and desolate landscapes, the Moon offers a unique glimpse into space exploration's past and future, making it a perfect destination for lunar adventurers.",
+    thumbnail: "/destination/image-moon.png",
+    isSelected: false,
+  },
+  {
+    name: "TITAN",
+    description:
+      "Titan, Saturn's largest moon, is a world of dense atmosphere and liquid methane lakes. This enigmatic moon is shrouded in a thick orange haze, concealing a landscape that is both alien and strangely familiar, beckoning explorers to uncover its secrets.",
+    thumbnail: "/destination/image-titan.png",
+    isSelected: false,
+  },
+];
 
 export const Destinations = () => {
-  const [selectedPlanets, onAddPlanet] = useState([]);
+  const [selectedPlanets, setSelectedPlanets] = useState([]);
 
-  let isPlanetSelected = false;
-  let numberOfPlanets = 0;
+  const onAddOrRemovePlanet = (name) => {
+    setSelectedPlanets((prevSelected) => {
+      if (prevSelected.includes(name)) {
+        return prevSelected.filter((planet) => planet !== name);
+      } else {
+        return [...prevSelected, name];
+      }
+    });
+  };
 
-  const onAddOrRemovePlanet = (name, index) => {
-    // TASK - React 1 week 2
-    // Implement this function
-    // If you press the "ADD PLANET" the selected planet should display "SELECTED"
-    // And the counter should update, how many planets are selected (numberOfPlanets)
-    console.log(`You seleceted the following planet: ${name}, with the index of ${index}`);
-  }
+  const numberOfPlanets = selectedPlanets.length;
+  const isSelected = (name) => selectedPlanets.includes(name);
 
   return (
     <div className="fullBGpicture">
       <main className="mainContent">
         <h1>Travel destinations</h1>
         <section className="card">
           <h2>Wishlist</h2>
-          {/* TASK - React 1 week 2 */}
-          {/* Display the number Of selected planets */}
-          {/* Display the "no planets" message if it is empty! */}
-          <p>No planets in wishlist :(</p>
-          <p>You have {numberOfPlanets} in your wishlist</p>
+
+          {numberOfPlanets === 0 ? (
```

Comment:
This is a very common practice all over the internet, just imagine opening your cart online and it will either:
- Display the items
- Have an icon and say your cart is empty